### PR TITLE
Fix #28

### LINKED ISSUE #28: Conflicts with 1.0.x libs
I `--force-latest`ed after `purescript-generics`, `purescript-integers`, `purescript-random`.

### PR DESCRIPTION
**Pull request by texastoland**
Perhaps `1.0.0-rc.1` until hdgarrood/purescript-sequences#23 drops.
- Update versions
- Comment out `ps-sequences`
- Nix deprecated `ps-arrows`
- Nix removed `Unsafe` modules
- Replace `Console.Unsafe` with `ps-debug`
- Replace `Aff.Par` with `ps-parallels` additions
- Replace `Contravariant.Divisible` with `ps-contravariant` additions
- Update broken modules esp. `ps-datetime`
- Update Prelude
- Remove bare parens from imports

### CODE REVIEW

**Comment by tfausak** [comment]
Clickable: #28.

**Comment by tfausak** [comment]
The build stalled, which doesn't necessarily represent a real failure. I noticed it used Node 5.x when it should really be using Node 6.x. 

Thanks for this! This is a ton of work. As we talked about, it would be nice to automate this at some point.

**Comment by texastoland** [comment]
Thanks! Test passed installing from scratch so I assumed Travis got mixed up.

File: src/Batteries.purs [hunk 1]
```diff
@@ -357,11 +337,7 @@ import Control.Monad.Aff.Class
   ( class MonadAff
   , liftAff
   )
--- import Control.Monad.Aff.Console -- NOTE: Math, Control.Monad.Eff.Console
-import Control.Monad.Aff.Par
-  ( Par(Par)
-  , runPar
-  )
+-- import Control.Monad.Aff.Console -- NOTE: Control.Monad.Eff.Console
```
Comment (texastoland) [inline src/Batteries.purs:340]: Note: it _should_ conflict with `eff` (whose functions were renamed in `0.8`) but _does_ conflict with `math`.

File: src/Batteries.purs [hunk 2]
```diff
@@ -1752,7 +1752,8 @@ import Optic.Laws.Lens
   , validLens
   )
 import Optic.Lens
-  ( lens
+  ( flip'
```
Comment (texastoland) [inline src/Batteries.purs:1755]: @st58 `flap` 😄

**Comment by tfausak** [comment]
The tests worked fine for me locally. I'm going to merge this and release it as 0.6.0.

**Pull request merged**